__tests__/app.test.js

```diff
@@ -166,4 +167,55 @@ describe.only('GET /api/articles/:article_id/comment/', () => {
             expect(res.status).toBe(400)
         })
     })
-})
\ No newline at end of file
+})
+
+describe.only('POST /api/articles/:article_id/comments', () => {
+    test('1. Returns comment', () => {
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send({ username: 'rogersop', body: 'test comment, from rogersop.'})
+    .then((res) => {
+        expect(201)
+        expect(res.body.comment).toEqual('test comment, from rogersop.')
+    })
+    })
+    test('2. Ignores extra properties on post body', () => {
+        return request(app)
+        .post('/api/articles/2/comments')
+        .send({ username: 'rogersop', body: 'test comment, from rogersop.', extra: 3})
+    .then((res) => {
+        expect(201)
+        expect(res.body.comment).toEqual('test comment, from rogersop.')
```

Comment:
You shouldn't be returning just the comment string. It should be the whole object as you may need access to the author in the future